server/src/main/java/org/apache/kafka/server/metrics/ClientMetricsInstance.java

```diff
@@ -116,8 +116,7 @@ public synchronized boolean maybeUpdatePushRequestTimestamp(long currentTime) {
         */
         boolean canAccept = lastGetRequestTimestamp > lastPushRequestTimestamp;
```

Comment:
how about `boolean canAccept = lastGetRequestTimestamp > lastPushRequestTimestamp || currentTime - lastPushRequestTimestamp >= pushIntervalMs;`?

Reply:
It's shorter but I'm not sure it's more readable.